src/main/java/org/tron/core/services/http/CreateAssetIssueServlet.java

```diff
@@ -0,0 +1,52 @@
+package org.tron.core.services.http;
+
+import java.io.IOException;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.tron.core.Wallet;
+import org.tron.core.exception.ContractValidateException;
+import org.tron.core.services.http.JsonFormat.ParseException;
+import org.tron.protos.Contract.AssetIssueContract;
+import org.tron.protos.Protocol.Transaction;
+import org.tron.protos.Protocol.Transaction.Contract.ContractType;
+
+
+@Component
+@Slf4j
+public class CreateAssetIssueServlet extends HttpServlet {
+
+  @Autowired
+  private Wallet wallet;
+
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+
+  }
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response) {
+    try {
+      String contract = request.getReader().lines()
+          .collect(Collectors.joining(System.lineSeparator()));
+      AssetIssueContract.Builder build = AssetIssueContract.newBuilder();
+      JsonFormat.merge(contract, build);
+      Transaction tx = wallet
+          .createTransactionCapsule(build.build(), ContractType.AssetIssueContract).getInstance();
+      response.getWriter().println(Util.printTransaction(tx));
+    } catch (ContractValidateException e) {
+      logger.debug("ContractValidateException: {}", e.getMessage());
+      try {
+        response.getWriter().println(Util.printErrorMsg(e));
+      } catch (IOException ioe) {
+        ioe.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1114&issues=AWRjYLfyMt9UOhrw8p6h&open=AWRjYLfyMt9UOhrw8p6h)

[ISSUE_KEY]: AWRjYLfyMt9UOhrw8p6h